packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/build-navigation-label.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+
+// Copied from packages/block-library/src/navigation/edit/navigation-menu-selector.js.
+export function buildNavigationLabel( title, id, status ) {
```

Comment:
For consistency should we use `default` export?

Reply:
I'm not a fan of using the default export as it doesn't enforce using the same name when it's imported, but happy to change it if you want.

Reply:
Updated.